src/fake_node_modules/powercord/components/ContextMenu/Submenu.jsx

```diff
@@ -84,6 +87,18 @@ class SubmenuItem extends React.Component {
     return submenu;
   }
 
+  getSubmenuImage () {
+    return (
+      this.props.image.startsWith('fa-')
+        ? <div style={{ cursor: 'pointer!important',
+          marginRight: '10px' }} class={`${this.props.image.endsWith('-alt')
+          ? 'far'
+          : 'fas'}
+          ${this.props.image.replace('-alt', '')} fa-fw`} />
+        : <img style={{ marginRight: '10px' }} src={this.props.image} />
```

Comment:
same, alt attribute is required on img tags in HTML specifications. Empty alt will do the trick